ec2/spark_ec2.py

```diff
@@ -349,6 +350,15 @@ def launch_cluster(conn, opts, cluster_name):
     if opts.identity_file is None:
         print >> stderr, "ERROR: Must provide an identity file (-i) for ssh connections."
         sys.exit(1)
+
+    if not os.path.exists(opts.identify_file):
+        print >> stderr, "ERROR: The identity file(%s) doesn't exist." % opts.identity_file
+        sys.exit(1)
+
+    if oct(os.stat(opts.identity_file).st_mode)[-3:] != "600":
```

Comment:
hi, nchammas. I would like to used "os.stat(xx).st_mode & stat.S_IRUSR" to check the identity file's permissions. 400 or 600 is a magic number, S_IRUSR is define in posix, to check weather owner has the read permissions.